go_client/getting_started.adoc

```diff
@@ -12,229 +12,29 @@ toc::[]
-client library and its dependencies will be included in your project.
+For client-go applications, you use a link:https://github.com/openshift/client-go[client library] and link:https://github.com/openshift/api[API object definitions] to create applications that interact with the OpenShift REST API. The openshift/client-go library is designed to be compatible with and complementary to the kubernetes/client-go library. To get started, we recommend you follow these general steps:
 
-The following example connects to {product-title} using a pre-existing
-kubeconfig file, then lists all pods and builds in the current namespace.  Copy
-it as main.go into a new directory under your GOPATH.
+1. **Read the link:https://github.com/kubernetes/client-go/blob/master/README.md[client-go README]**: This file helps you understand the components of Kubernetes client-go, as well as which client-go versions you should use with different versions of Kubernetes. You can also see link:https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration[in-cluster] and link:https://github.com/kubernetes/client-go/tree/master/examples/out-of-cluster-client-configuration[out-of-cluster] examples of go client applications. Refer to link:https://github.com/openshift/client-go[OpenShift Client in Go] for specifics about OpenShift compatibility issues.
 
-.$GOPATH/src/gettingstarted/main.go
-[source, go]
-----
-package main
+1. **Read the link:https://github.com/openshift/client-go/blob/master/INSTALL.md[client-go INSTALL]: This helps you set up your client-go environment by vendoring (downloading) the correct code and helps you understand the tools you need to create applications.  Dependency management in particular is addressed.
 
-import (
-	"fmt"
+1. **Review link:https://github.com/kubernetes/client-go/tree/master/examples[client-go examples]**: These include examples of cluster authentication and managing resources, as well as more advanced concepts for interfacing with work queues and custom resource definitions.
 
-	buildv1client "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+1. **Review link:https://github.com/openshift/client-go/tree/master/examples/build[client-go build operations]**: Demonstrates basic operations for using Build resources. These include List builds, Retrieve details of a build, and Trigger a build.
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-	"k8s.io/client-go/tools/clientcmd"
-)
+1. Navigate the link:https://github.com/openshift/client-go/[client-go source code]**: Go through the client-go source code, which contains the client-side code to invoke most actions against a cluster. {product-title} client-go is backwards compatible to OpenShift 3.6. To use the source code in a sample go project, all you need to do is to vendor these artifacts and write the code (when using Glide make sure to use 0.13).
```

Comment:
s/Navigate/Review
s/OpenShift 3.6/{product-title} version 3.6

"To use the source code in a sample go project, all you need to do is to vendor these artifacts and write the code (when using Glide make sure to use 0.13)." I think we need to provide more information about how to do this or remove the statement. If you keep it, you might need to say a bit more about Glide, since we don't discuss it here, and try not to use parentheses.